06. contribute/07. features.md

```diff
@@ -25,6 +25,3 @@ After you installed [Git](https://help.github.com/articles/set-up-git), start by
 After you [comitted](https://help.github.com/articles/create-a-repo#step-2-commit-your-readme) all changes to your branch and [pushed](https://help.github.com/articles/create-a-repo#step-3-push-your-commit) them to your forked version, you'll have to do a [pull request](https://help.github.com/articles/using-pull-requests). Make sure you explain what and why you want to change a part to the core. After the pull request has been voted or discussed on the [technical board meeting](index.md) it will get merged.
 
 More information about working with GitHub can found on the [help](https://help.github.com/) of GitHub, in the [documentation](http://git-scm.com/documentation) of Git, in the [Git Workflows](http://documentup.com/skwp/git-workflows-book) book or learn Git Branching [interactively](http://pcottle.github.io/learnGitBranching/).
-
-
-When you edit something in the core we will ask you to update the documentation too. Read our [documentation](03. documentation.md) article to find some tips.
```

Comment:
well, I think it would be a good idea to start doing this again instead of removing this